metastats-domain/src/main/groovy/life/qbic/metastats/datamodel/MetaStatsPackageEntry.groovy

```diff
@@ -0,0 +1,11 @@
+package life.qbic.metastats.datamodel
+
+class MetaStatsPackageEntry {
+    String entryId
+    HashMap properties
+
+    MetaStatsPackageEntry(String entryId, HashMap properties){
+        this.entryId = entryId
+        this.properties = properties
```

Comment:
Check if they are null before assigning

Reply:
I am not sure if I understand how you mean that. 
So you suggest I should do something like `if(entriID != null) this.entryID = entryID` ?